modules/snooker/src/main/ts/ephox/snooker/api/Structs.ts

```diff
@@ -74,6 +74,10 @@ export interface Column {
   readonly colspan: number;
 }
 
+export interface ColumnExt extends Column {
```

Comment:
Do we use `Column` anywhere, or can `ColumnExt` be folded into it? I've not seen anywhere where we just use `Column` except for on what's really `ColumnExt`. Did I miss it?

Reply:
It's not currently used, but we want to keep things consistent. As you'll see elsewhere it uses to `Detail` for core details and `DetailExt` for details with the row/column position.

Reply:
That seems like a strange reason to me. Can't you just call the first one ColumnExt to start with, to reduce confusion if that's the case?

Reply:
Consistency is always preferred and remember that `Column` might need to be used outside this code, as this is public APIs.

Reply:
Consistency is good but this seems like a strange reason and place for it. Nothing with `Row` in the name is Ext, only detail, which represents a cell. And I'll admit I don't see when they'd need to use `Column` instead of `ColumnExt`, except for us telling them that the type is `Column` so that's what they'll use.

Reply:
That's because a Row is not the equivalent of a Column. A Cell is the equivalent of that, so you're comparing an apple to an orange in the above.

Reply:
I noticed a few possibly duplicate types in Snooker while doing this - we should maybe just Jira up investigating that? Would be nice to simplify them if possible.

Reply:
With regards to the second point, a `Column` or `Detail` is just the details about the element. Whereas the `Ext` variants contain "extended" details about where the item is in the grid.

Reply:
I found some references to `Column` anyway, to answer Hannes's initial question.

Reply:
Okay, I missed it was used in `redistributeToColumns` sorry.